06_source/ADAS/ADAS_MCU/ADAS_Cfg.h

```diff
@@ -39,45 +53,89 @@
+/** Encoder Interrupt for Right Motor*/
 #define PIN_ENC_R                                        2
+/** Encoder Interrupt for Left Motor*/
 #define PIN_ENC_L                                        3
+/** External Start Button*/
 #define Btn_Start                                       46
+/** Voltage Level of Battery*/
 #define PIN_VBAT                                        A0
 
 
 //Navigation parameters
+/** Navigation parameter: Desired offset to wall on right side in cm. */
 #define NAV_SET_OFFSET                                  80  // in cm
+/** Navigation parameter: Allowed tolerance (+-) of offset to wall on right side in cm. */
 #define NAV_TOL_OFFSET                                  5   // in cm
+/** Navigation parameter: Allowed tolerance (+-) of angle in degrees */
 #define NAV_TOL_ANGLE_DRIVE                             5   // degrees
 
 
 //Inter-Controller Communication Protocol
+/** Inter-Controller Communication: Definition of first start byte */
 #define ICC_STX1                                        0xff
+/** Inter-Controller Communication: Definition of second start byte */
 #define ICC_STX2                                        0xff
+/** Inter-Controller Communication: Definition of stop byte */
 #define ICC_TTX                                         0xee
+/** Inter-Controller Communication: Definition of length of message (including STXs and TTX) */
 #define ICC_LEN                                         6U // including STXs and TTX
+/** Inter-Controller Communication: Command to drive continuous */
 #define ICC_CMD_CONT_DRIVE_IN                           0x01
+/** Inter-Controller Communication: Command to do an emergency stop */
 #define ICC_CMD_EMER_STOP                               0x02
+/** Inter-Controller Communication: Command to do a soft stop */
 #define ICC_CMD_SOFT_STOP                               0x03
+/** Inter-Controller Communication: Command to drive a distance */
 #define ICC_CMD_DRIVE_DIST                              0x04
+/** Inter-Controller Communication: Command to rotate by angle */
 #define ICC_CMD_ROT_ANGLE                               0x05
+/** Inter-Controller Communication: Command to set speed */
 #define ICC_CMD_SET_SPEED                               0x06
+/** Inter-Controller Communication: Command to pause movement */
 #define ICC_CMD_PAUSE_DRIVE                             0x07
+/** Inter-Controller Communication: Command to continue movement */
 #define ICC_CMD_CONT_DRIVE                              0x08
+/** Inter-Controller Communication: Command to acknowledge message */
 #define ICC_CMD_FB_ACK                                  0x01
+/** Inter-Controller Communication: Command to acknowledge reached distance */
 #define ICC_CMD_FB_DIST                                 0x02
+/** Inter-Controller Communication: Command to acknowledge reached rotation */
 #define ICC_CMD_FB_ROT                                  0x03
 
 //Environmetal parameters
```

Comment:
Environmetal parameters not needed on MCU.

Reply:
We are sending this to you as a feedback

Reply:
Ahh.. I see you have marked the wrong line so I was confused :-D removed it

Reply:
Yeah...sorry my mistake.